healvis/simulator.py

```diff
@@ -436,11 +573,13 @@ def run_simulation(param_file, Nprocs=1, sjob_id=None, add_to_history=''):
     # ---------------------------
     beam_attr = param_dict['beam'].copy()
     beam_type = beam_attr.pop("beam_type")
-    pols = beam_attr.pop("pols")
-    beam_freq_interp = beam_attr.pop("beam_freq_interp")
-    smooth_beam = beam_attr.pop("smooth_beam")
-    smooth_scale = beam_attr.pop("smooth_scale")
-    obs = setup_observatory_from_uvdata(uv_obj, fov=param_dict['beam'].pop("fov"), set_pointings=True,
+
+    pols = beam_attr.pop("pols", None)
+    beam_freq_interp = beam_attr.pop("beam_freq_interp", 'cubic')
+    smooth_beam = beam_attr.pop("smooth_beam", False)
+    smooth_scale = beam_attr.pop("smooth_scale", 2.0)
```

Comment:
Even though I agree that `2.0` is the default in the function, I think this should be `smooth_scale = beam_attr.pop("smooth_scale", 2.0)`, because if someone isn't smoothing it won't be used anyways, but if they _are_ smoothing and forgot to specify the scale, then this will raise an error downstream, as one would hope if they user forgot to specify a crucial parameter.